### ISSUE TITLE
[Manure] Implement New Separator Configs (Types)

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#1770]
Using the current implementation of Manure Separators, reach out to SMEs to get configs for the following separators:

- Weeping Wall
- Gravity Separator
- Settling Basin
- Centrifuge
- Evaporation
- Vermifiltration
- Sand Separator

And update the available manure_separator_configs in the manure_management.json input file (MASM > input > manure > manure_management.json)

This will include the following information as is outlined in the current configs for manure separators:
```
"manure_separator_configs": [
    {
      "name": "rotary screen",
      "manure_separator_type": "rotary screen",
      "percent_dry_solids": 0.20,
      "total_solids_removal_efficiency_for_separator": 0.35,
      "volatile_solids_removal_efficiency_for_separator": 0.40,
      "nitrogen_removal_efficiency_for_separator": 0.30,
      "total_ammoniacal_nitrogen_removal_efficiency_for_separator": 0.15,
      "phosphorus_removal_efficiency_for_separator": 0.40,
      "potassium_removal_efficiency_for_separator": 0.15
    },
    {
      "name": "screw press",
      "manure_separator_type": "screw press",
      "percent_dry_solids": 0.35,
      "total_solids_removal_efficiency_for_separator": 0.25,
      "volatile_solids_removal_efficiency_for_separator": 0.30,
      "nitrogen_removal_efficiency_for_separator": 0.30,
      "total_ammoniacal_nitrogen_removal_efficiency_for_separator": 0.10,
      "phosphorus_removal_efficiency_for_separator": 0.20,
      "potassium_removal_efficiency_for_separator": 0.23
    }
  ],
```

Note: there may not be sufficient research data available on some of these. If that is the case, make a future issue for implementation for those separator configs with whatever information/timeline is available from the SME(s).

**Comment by Isgotkowitz** [comment]
Grooming session: this issue will require a lot of back and forth with SMEs, so that should be accounted for when deciding how to fit this issue into workload.

**Comment by ew3361zh** [comment]
From meeting with Elle:
- Reiterate that this needs heavy SME input (not Elle) and is scientifically complex
- Her impression is this is not in high demand (priority is not Medium as is currently marked).

Resultingly, I will update this to be blocked.

**Comment by Isgotkowitz** [comment]
Note: Remain blocked as of 2/19/25.
Grooming session: this issue should be addressed after #2092 is completed

**Comment by allisterakun** [comment]
@elle-andreen is this a relevant issue post manure refresh?

**Comment by elle-andreen** [comment]
It is still a valid issue! But it would be a post-refresh item. It's still blocked by the need for SME input.

**Comment by elle-andreen** [comment]
7/7/25 post manure refresh update: still a valid issue but blocked by SME input.

**Comment by elle-andreen** [comment]
Also see issue #2509

**Comment by elle-andreen** [comment]
SME grooming 2/27/26: Still a valid issue but low priority for the moment and needs addition SME input/research.
